tools/ncpms-api-crawler/ncpms-crawler.py

```
@@ -0,0 +1,206 @@
+            if official_pest in res_pest or res_pest in official_pest:
+                fallback_key = item.get("insectKey", "")
+                break
+                
+        if fallback_key:
+            details = fetch_details(fallback_key)
+            results.append({
+                "pest_name": pest,
+                "crop_name": "fallback",
+                **details
+            })
+            print(f" -> 작물없음(Fallback) 추출 완료")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)